application/controllers/signup.php

```diff
@@ -0,0 +1,45 @@
+<?php 
+
+require_once './vendor/autoload.php';
+
+use Database\Database;
+use User\User;
+
+$Object_database = new Database();
+$Object_user = new User;
+
+
+if(isset($_POST['submit_register'])){
```

Comment:
In this page please make proper indentaion also check give space before `{` brace.